src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+        return formattedString;
+    }
+
+    private static void unmark(String line){
+        int input = Integer.valueOf(line.replace("unmark", "").strip());
+        Task task = inputs[input - 1];
+        task.markAsNotDone();
+        System.out.println(HORIZONTAL_LINE + "OK, I've marked this task as not done yet: \n  "
+                + task + "\n" + HORIZONTAL_LINE);
+    }
+
+    private static void mark(String line) {
+        int input = Integer.valueOf(line.replace("mark", "").strip());
+        Task task = inputs[input - 1];
+        task.markAsDone();
+        System.out.println(HORIZONTAL_LINE + "Nice! I've marked this task as done: \n  "
+                + task + "\n"+ HORIZONTAL_LINE);
+    }
+
+    private static void createTodo(String line){
+        String description = line.replace("todo", "").strip();
+        Todo todo = new Todo(description);
+        inputs[index] = todo;
+        index++;
+        System.out.println(HORIZONTAL_LINE + ADD_CONFIRMATION_MESSAGE + "\n  " + todo
+                + "\nNow you have "+ index + " tasks in the list.\n" + HORIZONTAL_LINE);
+    }
+
+    private static void createEvent(String line){
+        String removedCommand = line.replace("event", "");
+        String description = removedCommand.substring(0, removedCommand.indexOf("/at")).strip();
+        String date = removedCommand.substring(removedCommand.indexOf("/at") + "/at".length()).strip();
+        Event event = new Event(description, date);
+        inputs[index] = event;
+        index++;
+        System.out.println(HORIZONTAL_LINE + ADD_CONFIRMATION_MESSAGE + "\n  " + event
+                + "\nNow you have "+ index + " tasks in the list.\n" + HORIZONTAL_LINE);
+    }
+
+    private static void createDeadline(String line) {
+        String removedCommand = line.replace("deadline", "");
+        String description = removedCommand.substring(0, removedCommand.indexOf("/by")).strip();
+        String date = removedCommand.substring(removedCommand.indexOf("/by") + "/by".length()).strip();
+        Deadline deadline = new Deadline(description, date);
+        inputs[index] = deadline;
+        index++;
+        System.out.println(HORIZONTAL_LINE + ADD_CONFIRMATION_MESSAGE + "\n  " + deadline
+                + "\nNow you have "+ index + " tasks in the list.\n" + HORIZONTAL_LINE);
+    }
+
+    private static void createTask(String line) {
+        System.out.println(HORIZONTAL_LINE + "added: " + line + "\n" + HORIZONTAL_LINE);
+        inputs[index] = new Task(line);
+        index++;
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        String line;
```

Comment:
"Variables should be initialized where they are declared and they should be declared in the smallest scope possible." 
In this case this String `line` could be declared and initialized in the while loop below, in a slightly smaller scope.